src/NBomber/DomainServices/Validation.fs

```diff
@@ -22,6 +22,21 @@ let private isDurationOk (duration: TimeSpan) =
 let private isConcurrentCopiesOk (value: int) = 
     value >= 1
 
+let private getInvalidAssertionStepNames (scenario: Contracts.Scenario) =

```

Comment:
can you please create a module AssertionValidation 
and put all these functinons there
and you can use shorter naming
"getInvalidAssertions"